src/overlays/actors/ovl_En_Ginko_Man/z_en_ginko_man.c

```diff
@@ -352,66 +318,51 @@ void EnGinkoMan_DepositDialogue(EnGinkoMan* this, GlobalContext* globalCtx) { //
                 this->curTextId = 0x45A; // "All right, little guy, now I've got a total of [rupees] from you!"
             }
             break;
-        case 43: // "...You know, at this time of day there's a 4 Rupee service charge on withdrawals!"
+        case 0x477: // "...You know, at this time of day there's a 4 Rupee service charge on withdrawals!"
             func_801518B0(globalCtx, 0x471, &this->actor);
             this->curTextId = 0x471; // "Are you really withdrawing [rupees selected]? Y/n"
             this->serviceFee = globalCtx->msgCtx.unk1206C;
-            return;
-        case 45: // Well, are you gonna make a deposit?
+            break;
+        case 0x479: // Well, are you gonna make a deposit?
             func_801518B0(globalCtx, 0x44F, &this->actor);
             this->curTextId = 0x44F; // "All right! So..."
-            return;
-        default:
             break;
     } // end switch
 
-} // */
-#else
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_En_Ginko_Man_0x80A644A0/EnGinkoMan_DepositDialogue.asm")
-#endif
+}
 
-#if NON_MATCHING
-// NON-MATCHING: lots of regalloc
-// ROM SHIFT: a couple redundant li 0x4806 added reason unk
 // actionfunc: wait for player dialogue input
 void EnGinkoMan_WaitForDialogueInput(EnGinkoMan* this, GlobalContext* globalCtx) {
-
-    // branch likely instead of branch, fixed by adding a fake if later though
-    // probably because its supposed to wrap around the whole switch, but couldn't match with that either
     if (func_80147624(globalCtx) == 0) {
         return;
     }
 
     switch (this->curTextId) {
-
         case 0x44E: // "...So, what'll it be?
             if (globalCtx->msgCtx.choiceIndex == GINKOMAN_CHOICE_YES) {
                 if ((gSaveContext.roomInf[127][0] & 0xFFFF) >= 0x1388) {
                     play_sound(0x4806); // NA_SE_SY_ERROR
                     func_801518B0(globalCtx, 0x45F, &this->actor);
                     this->curTextId = 0x45F; // bank full, cannot accept more
-                    return;
                 } else {
                     if (gSaveContext.rupees > 0) {
-                        func_8019F208(0x4806);
+                        func_8019F208();
```

Comment:
It was right in front of my face....

I wonder if mips2c added that as a parm or if I did, I don't even remember...

Reply:
I suspect it was mips2c since it would have stored `0x4806` in register `a0` for `play_sound(0x4806)`, and a combination of that value still being in `a0` and `func_8019F208()` not being forward-declared when you did this PR would have cause mips2c to put that argument in